core/include/traccc/edm/seed.hpp

```diff
@@ -46,56 +46,60 @@ struct seed {
         return *this;
     }
 
-    TRACCC_HOST_DEVICE
+    TRACCC_HOST
     std::vector<measurement> get_measurements(
         const host_spacepoint_container& spacepoints) const {
         std::vector<measurement> result;
+        result.reserve(3);
         result.push_back(spacepoints.at(spB_link).meas);
         result.push_back(spacepoints.at(spM_link).meas);
         result.push_back(spacepoints.at(spT_link).meas);
         return result;
     }
+
+    TRACCC_HOST
+    std::vector<spacepoint> get_spacepoints(
+        const host_spacepoint_container& spacepoints) const {
```

Comment:
:thinking: How would you feel about using `std::array<measurement, 3>` and `std::array<spacepoint, 3>` objects instead?

Reply:
Makes sense.
I was taking account of the case where we have different size of seeds, but I don't think we will do such crazy thing.